packages/core/src/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -0,0 +1,152 @@
+import { Component, h, Prop, Watch, Element } from "@stencil/core";
+import clsx from "clsx";
+import { tabbable } from "tabbable";
+import { v4 as uuidv4 } from "uuid";
+
+@Component({
+  tag: "dso-dropdown-menu",
+  shadow: true,
+})
+export class DropdownMenu {
+  /**
+   * Whether the menu is open or closed.
+   * This attribute is reflected and mutable.
+   */
+  @Prop({ reflect: true, mutable: true })
+  open = false;
+
+  /**
+   * Alignment of the dropdown
+   */
+  @Prop()
+  dropdownAlign: "left" | "right" = "left";
```

Comment:
Ik ben niet overtuigd van de naam van deze property: `<dso-dropdown-menu dropdown-align="...">` is dubbelop, we zitten al in de context van dropdown. De `dropdown-` prefix is wat mij betreft niet nodig.

Ik stel voor: `<dso-dropdown-menu align="...">`

Of had jij nog een gedachte achter jouw voorstel?

Reply:
`align=` was ook het eerste wat er in mij op kwam, echter, dat werkt ook meteen al als align op het component element (iig in Chrome) dus dat leek mij niet zo'n heel goed idee. Vandaar de 'overbodige' prefix.

Reply:
🤯 Ik had de HTML5 typings in VSCode bekeken en kwam alleen `align` op table en dergelijke tegen.. Natuurlijk dat de browser ook nog iets bedacht heeft..

Wat denk je dan van `align-menu="left"` oid?

Of `dso-align`.. Die klinkt raar maar Stencil weigert bv. een custom event een bestaande naam zoals `click` te maken. Daarom zie in de community dat de project prefix wordt gebruikt: `dsoChange` of `dsoClick`.